packages/tasks/src/local-apps.ts

```diff
@@ -99,6 +99,18 @@ const snippetLlamacpp = (model: ModelData, filepath?: string): LocalAppSnippet[]
 	];
 };
 
+const snippetLocalAI = (model: ModelData, filepath?: string): string[] => {
+	return [
+		`# Option 1: use LocalAI with docker
```

Comment:
One last comment on this:

Let's switch the order of the snippets, as I assume the number of people who will use this with docker are far less than directly from the binaries.

Question for @enzostvs - do you think doing something like we did for llama.cpp here: https://github.com/huggingface/huggingface.js/blob/3a8651fbc6508920475564a692bf0e5b601d9343/packages/tasks/src/local-apps.ts#L65 (since we have two snippets make sense)

Reply:
inverted as requested in https://github.com/huggingface/huggingface.js/pull/833/commits/3fca3d6b1bcedbd1306eedbf4269667f360beae4 !

Reply:
Perfect! @mudler sorry for the back and forth, do you think you'll be able to split the snippets the way I linked above? This way the users will have a much better `readability` experience on the hub.

Reply:
> Perfect! @mudler sorry for the back and forth, do you think you'll be able to split the snippets the way I linked above? This way the users will have a much better `readability` experience on the hub.

I can try, javascript isn't really my cup of tea :sweat_smile:

Reply:
Updated in https://github.com/huggingface/huggingface.js/pull/833/commits/1d5064ce6335f91841b8d4cf6816c31dc05061c0 - let me know if that looks good!